.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:7.10
```

Comment:
**SUGGESTION**: I think we can use the LTS version of node, right now is the version `8.9.3`